Oomd.h

```diff
@@ -78,6 +78,10 @@ class Oomd {
   std::vector<std::unique_ptr<Cgroup>> cgroups_;
 
   bool warned_io_pressure_{false};
+  /*bool warned_memcurrent{false};
```

Comment:
Commiting commented out code is generally a bad idea. Such code is not compiled or tested and could be invalid by the time someone tries to uncomment.